airy/airy.go

```diff
@@ -0,0 +1,37 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
```

Comment:
Ah, shoot. My fortran compiler isn't linking in at the moment, so I thought I could shortcut.